src/AnnenErfaring/Seksjon.elm

```diff
@@ -0,0 +1,1025 @@
+    }
+
+
+meldingsLogg : Model -> MeldingsLogg
+meldingsLogg (Model model) =
+    model.seksjonsMeldingsLogg
+
+
+type SamtaleStatus
+    = IkkeFerdig ( Model, Cmd Msg )
+    | Ferdig (List AnnenErfaring) FerdigAnimertMeldingsLogg
+
+
+type Samtale
+    = RegistrerRolle RolleInfo
+    | RegistrerBeskrivelse BeskrivelseInfo
+    | RegistrerFraMåned FraDatoInfo
+    | RegistrerFraÅr FraDatoInfo
+    | RegistrerNåværende ValidertFraDatoInfo
+    | RegistrerTilMåned TilDatoInfo
+    | RegistrerTilÅr TilDatoInfo
+    | VisOppsummering ValidertAnnenErfaringSkjema
+    | EndreOpplysninger AnnenErfaringSkjema
+    | VisOppsummeringEtterEndring ValidertAnnenErfaringSkjema
+    | LagrerSkjema ValidertAnnenErfaringSkjema
+    | LagringFeilet Http.Error ValidertAnnenErfaringSkjema
+    | VenterPåAnimasjonFørFullføring (List AnnenErfaring)
+
+
+type alias RolleInfo =
+    { rolle : String
+    , visFeilmeldingRolle : Bool
+    }
+
+
+type alias BeskrivelseInfo =
+    { rolle : String
+    , beskrivelse : String
+    }
+
+
+type alias FraDatoInfo =
+    { rolle : String
+    , beskrivelse : String
+    , fraMåned : Måned
+    , fraÅr : String
+    , visFeilmeldingÅr : Bool
+    }
+
+
+type alias ValidertFraDatoInfo =
+    { rolle : String
+    , beskrivelse : String
+    , fraMåned : Måned
+    , fraÅr : År
+    }
+
+
+type alias TilDatoInfo =
+    { rolle : String
+    , beskrivelse : String
+    , fraMåned : Måned
+    , fraÅr : År
+    , tilMåned : Måned
+    , tilÅr : String
+    , visFeilmeldingÅr : Bool
+    }
+
+
+rolleTilBeskrivelse : String -> BeskrivelseInfo
+rolleTilBeskrivelse rolle =
+    { rolle = rolle
+    , beskrivelse = ""
+    }
+
+
+beskrivelseTilFraDato : BeskrivelseInfo -> FraDatoInfo
+beskrivelseTilFraDato input =
+    { rolle = input.rolle
+    , beskrivelse = input.beskrivelse
+    , fraMåned = Januar
+    , fraÅr = ""
+    , visFeilmeldingÅr = False
+    }
+
+
+fraDatoTilTilDato : ValidertFraDatoInfo -> TilDatoInfo
+fraDatoTilTilDato input =
+    { rolle = input.rolle
+    , beskrivelse = input.beskrivelse
+    , fraMåned = input.fraMåned
+    , fraÅr = input.fraÅr
+    , tilMåned = Januar
+    , tilÅr = ""
+    , visFeilmeldingÅr = False
+    }
+
+
+fraDatoTilSkjema : ValidertFraDatoInfo -> ValidertAnnenErfaringSkjema
+fraDatoTilSkjema info =
+    Skjema.initValidertSkjema
+        { rolle = info.rolle
+        , beskrivelse = info.beskrivelse
+        , fraMåned = info.fraMåned
+        , fraÅr = info.fraÅr
+        , tilDato = Nåværende
+        , id = Nothing
+        }
+
+
+tilDatoTilSkjema : TilDatoInfo -> År -> ValidertAnnenErfaringSkjema
+tilDatoTilSkjema info tilÅr =
+    Skjema.initValidertSkjema
+        { rolle = info.rolle
+        , beskrivelse = info.beskrivelse
+        , fraMåned = info.fraMåned
+        , fraÅr = info.fraÅr
+        , tilDato = Avsluttet info.tilMåned tilÅr
+        , id = Nothing
+        }
+
+
+
+--- UPDATE ---
+
+
+type Msg
+    = VilRegistrereRolle
+    | OppdaterRolle String
```

Comment:
Kanskje endre til "OppdatererRolle" her, sånn Msg-en beskriver hva som har skjedd, i stedet for hva man vil skal skje? Samme med de andre oppdaterings-Msg-ene nedover